shared/hooks/useSearch.ts

```diff
@@ -22,6 +23,18 @@ export const prepareSearchDocuments = (
 // Modifying single documents or replacing documents with a same length array will therefore NOT trigger an update
 const useSearch = (documents: SearchResult[], query: string): SearchResult[] => {
   const [indexing, setIndexing] = useState(false)
+  const [debouncedQuery, setDebouncedQuery] = useState(query)
```

Comment:
:x: It seems to me like this is never actually used. Did you forget to use it in L61?

Reply:
:see_no_evil: yes